src/script/service/CoreSingleton.ts

```diff
@@ -62,6 +62,7 @@ export class Core extends Account {
           ? {
               keyingMaterialUpdateThreshold: Config.getConfig().FEATURE.MLS_CONFIG_KEYING_MATERIAL_UPDATE_THRESHOLD,
               defaultCiphersuite: Config.getConfig().FEATURE.MLS_CONFIG_DEFAULT_CIPHERSUITE,
+              useE2EI: Config.getConfig().FEATURE.ENABLE_E2EI,
```

Comment:
```suggestion
              enableE2EI: Config.getConfig().FEATURE.ENABLE_E2EI,
```

Would probably be semantically a little more precise. It's about enabling a feature, not using something in particular, I believe

Reply:
I dont think the same way. 

Since there is no way to really disable the feature, I wanted to use something other than the word enable.

But I can change that